### ISSUE TITLE
Experimental keys on scalar is now forbidden

### ISSUE THREAD

**Issue by faraco** [faraco/WebService-Gitter#1]
There's a compilation error with perl 5.24.0 and newer:
Occured in `v0.0.1`
```perl
#   Failed test 'use WebService::Gitter;'
#   at t/00_load.t line 5.
#     Tried to use 'WebService::Gitter'.
#     Error:  Experimental keys on scalar is now forbidden at /home/cpansand/.cpan/build/2017011806/WebService-Gitter-0.2.0-SG8Yvd/blib/lib/WebService/Gitter.pm line 37.
# Compilation failed in require at t/00_load.t line 5.
# BEGIN failed--compilation aborted at t/00_load.t line 5.
# Looks like you failed 1 test of 3.
t/00_load.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```

Original requestor by - SREZI@cpan.org
Assigned to - faraco@cpan.org

Rt ticket - https://rt.cpan.org/Ticket/Display.html?id=119894

**Comment by faraco** [comment]
Releasing fixes at patch v0.2.1

**Comment by faraco** [comment]
Fixed at version v0.3.0
